src/main/java/de/vill/model/LanguageLevel.java

```diff
@@ -13,14 +13,14 @@ public enum LanguageLevel {
     // MAJOR LEVELS (logic: val % 2 != 0)
     BOOLEAN_LEVEL(1, Constants.BOOLEAN_LEVEL),
     ARITHMETIC_LEVEL(3, Constants.ARITHMETIC_LEVEL),
-    TYPE_LEVEL(5, Constants.TYPE_LEVEL),
+    TYPE_LEVEL(7, Constants.TYPE_LEVEL),
```

Comment:
Can you explain the idea behind this change?

Reply:
Before the change, all minor language levels had the same priority. Meaning it did not matter in which order they were converted. However, for feature cardinality and aggregate function it does make a difference in which order they are converted. In the implementation major levels have an odd number and minor levels an even number. If Arithmetic level is 3and typelevel is 5, there is only one even number in between. To give aggregate function and type level different prioritys (give them different even numbers), I increased the value of the type level. If there are more typelevels added to UVL this rather simple approach of even and odd numbers should maybe be refactored.